po/zh.po

```diff
@@ -71,58 +71,58 @@ msgstr "从<配置文件>里读取配置信息"
 
 #: newsboat.cpp:69
 msgid "compact the cache"
-msgstr ""
+msgstr "压缩缓存"
 
 #: newsboat.cpp:73
 msgid "<command>..."
-msgstr "命令 ..."
+msgstr "<命令> ..."
 
 #: newsboat.cpp:74
 msgid "execute list of commands"
 msgstr "执行一系列命令"
 
 #: newsboat.cpp:76
 msgid "quiet startup"
-msgstr ""
+msgstr "启动时不输出信息"
 
 #: newsboat.cpp:77
 msgid "get version information"
 msgstr "获得版本信息"
 
 #: newsboat.cpp:81 src/pbcontroller.cpp:348
 msgid "<loglevel>"
-msgstr "记录等级"
+msgstr "<日志等级>"
 
 #: newsboat.cpp:82
 #, fuzzy
```

Comment:
Since you addressed this translation, you can remove this fuzzy marker and all the other ones.